template/workflows/helm/.github/workflows/azure-kubernetes-service-helm.yml

```diff
@@ -45,13 +46,14 @@ env:
   AZURE_CONTAINER_REGISTRY: {{ .Config.GetVariableValue "AZURECONTAINERREGISTRY" }}
   CONTAINER_NAME: {{ .Config.GetVariableValue "CONTAINERNAME" }}
   CLUSTER_RESOURCE_GROUP: {{ .Config.GetVariableValue "CLUSTERRESOURCEGROUP" }}
-  CLUSTER_NAME: {{ .Config.GetVariableValue "CLUSTERNAME" }}
+  CLUSTER_RESOURCE_TYPE: Microsoft.ContainerService/{{ .Config.GetVariableValue "RESOURCETYPE" }}
```

Comment:
Can you remind me again where we set the value of the config to default to managedCluster?  By defaulting to managedCluster, we can ensure backward compatibility.